### PR TITLE
1.6 deployment updates

### PR DESCRIPTION
**Pull request by Hwesta**
Updates to match suggestions in artefactual/archivematica-docs#17

* Add an Archivematica 1.6 install file
* Default to production installation, set development in the qa vars file
* Default to 1.6 installation, not dev

### CODE REVIEW

**Comment by hakamine** [comment]
:+1: Looks good! Once the stable/1.6.x and stable/0.10.x branches in the AM and SS repos are created I think this is good to go.
Also, would it be better if the `include_vars` in `singlenode.yml` is left assigned `vars-singlenode-qa.yml`? (so that the instructions in https://github.com/artefactual/archivematica-docs/pull/17/files do not need to be changed) (or we just need to remember to change the docs)

**Comment by hakamine** [comment]
Found something missing, I think that in vars-singlenode-1.6.yml  the following should be added , in order to use the externals repo corresponding to 1.6: 
```
archivematica_src_externals_repo: "http://packages.archivematica.org/1.6.x/ubuntu-externals" 
```

**Comment by Hwesta** [comment]
> Also, would it be better if the include_vars in singlenode.yml is left assigned vars-singlenode-qa.yml? (so that the instructions in https://github.com/artefactual/archivematica-docs/pull/17/files do not need to be changed) (or we just need to remember to change the docs)

Do we want the default deploy-pub install to be production (with the latest stable release & production flag), or development (with qa & development flag)?  In the same way the default branch in a repo is the latest stable branch, I think the default install should be the latest stable.  I wrote the docs for the way it was, but hoping they would be changed so I could remove the file-editing instructions. Thoughts?

File: playbooks/archivematica/Vagrantfile [hunk 1]
```diff
@@ -39,7 +39,6 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     ansible.host_key_checking = false
     ansible.extra_vars = {
       "archivematica_src_dir" => "/vagrant/src",
-      "archivematica_src_environment_type" => "development",
```
Comment (jhsimpson) [inline playbooks/archivematica/Vagrantfile]: I think this line could be left in.  The Vagrantfile is meant to set up a development environment.  Maybe this file should override archivematica_src_am_version and archivematica_src_ss_version as well, to point at qa branches?  

The readme.md in this repo could be clear that 'vagrant up' will create a dev environment, and for production uses it is necessary to have a hosts file and do ansible-playbook commands instead.  When you use ansible-playbook, the Vagrantfile is ignored, and the extra_vars in there are not used.

**Comment by Hwesta** [comment]
After discussion, removed the change to the Vagrantfile, since a Vagrant deploy is always development.  Keeping the vars-singlenode-1.6.yml and setting it as the default, for the same reasons that the default branch in git is the latest stable release.

**Pull request merged**